home/ivp-planning/advanced-tools/path-tool.md

```diff
@@ -1,102 +1,80 @@
+To complete a path, you can either press **Enter** or **double-click on the last measurement point** set. Upon completion, the path is saved as a permanent object that can be adjusted or deleted in the [hierarchy panel](../user-interface/hierarchy-panel.md)
 
 {% hint style="info" %}
 If the path contains three or more measurement points, it will automatically be closed after creation (the first and the last measurement points will be connected automatically).
 {% endhint %}
 
-![](../../../.gitbook/assets/iVP\_path\_tool\_draw\_path\_path\_panel.jpg)
+![](../../../.gitbook/assets/iVP_path_tool_draw_path_panel_new.jpg)
 
-The path creation can be cancelled at any time by pressing the **Escape key**. This can be used to quickly measure distances or angles without creating a permanent path.
+The path creation can be canceled at any time by pressing the **Escape key**. This allows for quick measurement of distances or angles without creating a permanent path.
 
-## Path selection and editing:
+## Path selection and editing
 
-Existing paths can be selected by **clicking on the path name** in the [2D-panel](../user-interface/the-2d-panel.md).
+Existing paths can be selected by **clicking on the path name** in the [hierarchy panel](../user-interface/hierarchy-panel.md).
 
-{% hint style="info" %}
-It is not possible to select paths by clicking on the path itself - this limitation is necessary to prevent unwanted modifications when working with overlapping paths.
-{% endhint %}
-
-![](../../../.gitbook/assets/iVP\_path\_tool\_path\_selection.jpg)
-
-To add new points to a selected path, **right click on a line** and subsequently click on **"Add Point"**.
+![](../../../.gitbook/assets/iVP_path_tool_path_selection_new.jpg)
 
-![](../../../.gitbook/assets/iVP\_path\_tool\_draw\_path\_right\_click\_line\_menu.jpg)
+To add new points to a selected path, **right-click on a line** and then click on **add point**.
+![](../../../.gitbook/assets/iVP_path_tool_draw_path_right_click_line_new.jpg)
 
 To delete a point, **right click on the point** and then click on **"Delete"**.
 
-![](../../../.gitbook/assets/iVP\_path\_tool\_draw\_path\_right\_click\_point\_menu.jpg)
-
-To move points it is enough to **drag them** with the **left mouse button**.
-
-## Measurement display options:
-
-The length and angle information of each path can be turned on and off individually by **right clicking on them** and choosing **"Length"** respectively **"Angle"** from the appearing menus.
-
-![](../../../.gitbook/assets/iVP\_path\_tool\_draw\_path\_right\_click\_line\_menu.jpg)
+![](../../../.gitbook/assets/iVP_path_tool_delete_point.jpg)
 
-![](../../../.gitbook/assets/iVP\_path\_tool\_draw\_path\_right\_click\_point\_menu.jpg)
+To move points, simply **drag them** with the **left mouse button**.
```

Comment:
Should mention that path points can be selected individually and moved in the 3D view